src/components/AttachmentModal.tsx

```diff
@@ -471,6 +473,12 @@ function AttachmentModal({
                     setShouldLoadAttachment(false);
                 }}
                 propagateSwipe
+                initialFocus={() => {
+                    if (!submitRef.current) {
+                        return false;
+                    }
+                    return submitRef.current as View | HTMLElement;
```

Comment:
Why is there a need to typecast here? Wouldn't `return submitRef.current;` work?